AppDevAnnouncements/Classes/Announcement.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Announcement.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 10/16/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+struct Announcement: Codable {
+
+    /// The URL of the image
+    let imageUrl : String?
+
+    /// The header title of the notification
+    let subject : String
+
+    /// The main text of the notification
+    let body : String
+
+    /// CTA - Call To Action
```

Comment:
Is CTA always gonna be a button? If so, can we make this comment explicitly mention that ctaText will be the button text